### ISSUE TITLE
Cannot seem to get SourceMod 1.10 working on the server

### ISSUE THREAD

**Issue by NoShenGG** [RGLgg/server-resources-updater#69]
Probably not the right place to post this, but I am attempting to run SourceMod 1.10 on my Linux-based server as per the Installation Instructions. I am grabbing the last build made on the 1.10 branch for SourceMod and the latest MetaMod stable build (1.11). When booting the server, it runs into a segmentation fault and crashes. I can provide more specific info if needed, but it appears to be that SourceMod 1.10 is not compatible with at least my server. My server is running on Ubuntu 22.04 on a droplet on DigitalOcean to provide some basic background info.

**Comment by l-Aad-l** [comment]
It could be because you have two different versions of Sourcemod/Metamod(1.10 + 1.11), we haven't tested with different versions of those plugins together since people tend to use matching major versions of these plugins. You could try to use 1.11 of Sourcemod and see if that works as well, I have heard rumors but no real confirmation of the prebuilt RGL plugins found under the releases functioning on SM 1.11.

**Comment by NoShenGG** [comment]
I just tried same versions of Sourcemod & Metamod together and got the same error when loading server up. When trying 1.11 of both of them, it seems fine. I run into issues with massive server lag when executing the rgl_6s_5cp_scrim config file. Not sure if that's just a limit of my machine that I'm using or if this is the "plugin breaking" as described.

**Comment by NoShenGG** [comment]
More accurately, the server framerate seems to stutter heavily on SM 1.11 when executing any of the RGL configs. Gameplay seems fine for the most part before actually executing any configs.

**Comment by l-Aad-l** [comment]
> More accurately, the server framerate seems to stutter heavily on SM 1.11 when executing any of the RGL configs. Gameplay seems fine for the most part before actually executing any configs.

This is normal when executing any TF2 configuration, it has nothing to do with the plugins installed, simply due to TF2 loading all the configuration changes.

**Comment by NoShenGG** [comment]
Even after the config is done executing? Only after the config is done executing does the server framerate stutter.

**Comment by l-Aad-l** [comment]
Yup it takes a few seconds for everything to load, if it persists after everything has loaded, then it's likely due to your server

**Comment by NoShenGG** [comment]
Upon further inspection, I have tried to load different parts of the configs manually depending on my personal judgment what would be most likely to cause server framerate issues. I believe the issue that I have is with the enabling of SourceTV. For what reason, I have no clue. I sincerely appreciate all the help you've given me so far, but I have to ask if there is a better place to direct my issues to considering I believe I have found the source of my problems.
